test/snippet/range/views/minimiser.cpp

```diff
@@ -18,4 +19,22 @@ int main()
 
     // kmer_hash with gaps, hashes: [2,7,8,14,1], minimiser: [2,1]
     seqan3::debug_stream << (text | seqan3::views::kmer_hash(0b101_shape) | seqan3::views::minimiser(4)) << '\n';
+
+    /* Minimiser view with two ranges
+     * The second range defines the hash values from the reverse complement, the second reverse is necessary to put the
+     * hash values in the correct order. For the example here:
+     * ACGTAGC | seqan3::views::complement                      => TGCATCG
+     *         | std::views::reverse                            => GCTACGT
+     *         | seqan3::views::kmer_hash(seqan3::ungapped{3})  => [39 (for GCA), 28 (for CTA), 49 (for TAC),
+     *                                                              6 (for ACG), 27 (for CGT)]
+     * "GCA" is not the reverse complement from the first k-mer in "ACGTAGC", which is "ACG", but "CGT" is.
+     * Therefore, a second reverse is necessary to find the smallest value between the original sequence and its
+     * reverse complement.
+     */
+    auto hashes2 = text | seqan3::views::complement | std::views::reverse
```

Comment:
```suggestion
    constexpr auto reverse_complement = std::views::reverse | seqan3::views::complement;
    auto hashes2 = text | reverse_complement
```

Reply:
So should I delete the example? Because I think I added it because it was requested from the reviewer before.
And yes, the minimiser_hash view is going to do the reverse complement stuff internally.

Reply:
I would leave it as part of the seqan3::detail::minimiser_view documentation and adapt it to use the constructor instead of the view.

Reply:
How do I put only a snippet/a documentation in the seqan3::detail::minimiser_view documentation?